### DISCUSSION TITLE
Specify a version of iOS to run tests using scan

### DISCUSSION THREAD

**Discussion by dylanmoo** [fastlane/fastlane#18940] [Ideas]
I've been trying to test my app on multiple versions of iOS, but with the current parameters I'm only able to specify the device type. E.g. `scan(devices: ["iPhone 8", "iPhone 12 Pro"])`

It would be great if scan supported specifying a version of iOS as one of the parameters like: `scan(os: "14.2")`

In the meantime, has anyone found a workaround for this?

**Comment by lucgrabowski** [comment]
Could you try going in Xcode to Window - devices and simulators and add a new simulator? You will be able to select device, os and name for it and later you should be able to use it by it's name on devices list in scan.

**Comment by vrutberg** [comment]
FWIW, I believe this issue asks the exact same question: https://github.com/fastlane/fastlane/issues/10409
